notebooks/03_evaluate/evaluation.ipynb

```diff
@@ -48,8 +48,8 @@
      "name": "stdout",
```

Comment:
<p>I think it would be good to cite these papers, I think you have previously shared them right?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>